build.gradle

```diff
@@ -46,6 +46,17 @@ def variants = ['mysql-tomcat', 'mariadb-tomcat', 'postgres-tomcat']
 //   - postgresql: https://github.com/xwiki/xwiki-platform/blob/master/pom.xml#L119 (link for master)
 
 def tokens = [
+    '17': [
+        xwikiVersion: '17.0.0',
+        xwikiSha256: 'fd4d25b42c5645d87f7ed242967161ccb2648688948de93649a5ca11a1845c34',
+        mysqlJDBCVersion: '9.1.0',
+        mysqlJDBCSha256: '8776e2ebc46072c9a47ea59d98298c4273bd9f16a7b26b5dfa4744535aa26c62',
+        mariadbJDBCVersion: '3.5.1',
+        mariadbJDBCSha256: '50a50c4a3c13c30dfbd40587f7ad9a496197d285ede0948641d9eee68fdf2106',
+        postgresJDBCVersion: '42.7.4',
+        postgresJDBCSha256: '188976721ead8e8627eb6d8389d500dccc0c9bebd885268a3047180274a6031e',
+        tomcatImageTag: '10.1-jdk21'
```

Comment:
New key to define the tag for tomcat images. It's optional and defaults to `9-jre17`

Reply:
Might be more sense to invert the logic (since `9-jre17` is kind of a legacy kept for old branches).

Reply:
Going that way, we can even make the parameter mandatory and make it explicit for all version, as we do already for many other dependencies. WDYT?

Reply:
Making mandatory is fine too I guess.

Reply:
Now maybe have a variable for the tomcat version and one for the java version would make it a bit easier to maintain.